.github/workflows/triage.yml

```diff
@@ -209,7 +209,7 @@ jobs:
               keep_if_no_match: true
 
             - label: CI-linux-self-hosted
-              path: Formula/(envoy|freetype|gdbm|harfbuzz|libssh2|libxcrypt|libxml2|llvm|openssl@1.1|python@3.11|qt(@5)?|xz|zstd).rb
```

Comment:
Shouldn't this be replaced by v3?

Reply:
#138216 finished in 5h24m so I think we're good for now.